extensions/2.0/Vendor/EXT_mesh_gpu_instancing/README.md

```diff
@@ -18,9 +18,11 @@ Written against the glTF 2.0 spec.
 
 ## Overview
 
-This extension is specfically designed to enable GPU instancing which renders many copies of a single mesh at once using a small number of draw calls.  It's useful for things 
-like trees, grass, road signs, etc.  The TRANSLATION, ROTATION, and SCALE attributes allows the mesh to be displayed at many different locations with different rotations and scales.  
-Custom attributes can use the underscore mechanism to achieve other effects (i.e. _ID, _TRANSFORM4x3, etc.).
+This extension is specifically designed to enable GPU instancing, rendering many copies of a single mesh at once using a small number of draw calls.  It's useful for things 
+like trees, grass, road signs, etc.  The `TRANSLATION`, `ROTATION`, and `SCALE` attributes allow the mesh to be displayed at many different locations, with different rotations and scales.  
+As with vertex attributes, custom instance attributes may be prefixed with an underscore (e.g. `_ID`, `_TRANSFORM4x3`, etc.) and used for application-specific effects.
```

Comment:
#1820 (which will definitely conflict with this so we need to merge one of these first...) removes _TRANSFORM4x3 on the grounds that it's redundant wrt POSITION etc. and as such doesn't seem like a good example (that, and there's no 4x3 type for matrices in glTF specification)

Reply:
Agreed, and updated!